metrics/graphite/graphite.go

```diff
@@ -0,0 +1,295 @@
+// Package graphite implements a graphite backend for package metrics.
+//
+// The current implementation ignores fields.
+package graphite
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
```

Comment:
stdlib package log is verboten ;) If your emitter will log, please take a logger in your constructor.